lib/account.js

```diff
@@ -280,12 +280,23 @@ class Account {
             actions: [transaction_1.functionCall(methodName, args, gas || constants_1.DEFAULT_FUNCTION_CALL_GAS, amount)]
         });
     }
-    functionCallV2({ contractId, methodName, args = {}, gas = constants_1.DEFAULT_FUNCTION_CALL_GAS, attachedDeposit, walletMeta, walletCallbackUrl, stringify }) {
+    functionCallV2({ contractId, methodName, args = {}, gas = constants_1.DEFAULT_FUNCTION_CALL_GAS, attachedDeposit, walletMeta, walletCallbackUrl, stringify, jsContract }) {
         this.validateArgs(args);
-        const stringifyArg = stringify === undefined ? transaction_1.stringifyJsonOrBytes : stringify;
+        let functionCallArgs;
+        if (jsContract) {
+            function encodeCall(contract, method, args) {
```

Comment:
If this is exported in `src.transaction.ts`, do we need to redefine it here?

Reply:
It's not redefined here. `src.transaction.ts` exports the default method for handling(stringifying) `args`. This only applies it if theres no custom handler passed.

Reply:
Ah ok thanks - Im not amazing at typescript